app/src/components/dataset/DatasetPicker.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { graphql, useLazyLoadQuery } from "react-relay";
+
+import { Item, Picker, PickerProps } from "@arizeai/components";
+
+import { DatasetPickerQuery } from "./__generated__/DatasetPickerQuery.graphql";
+
+type DatasetPickerProps = Pick<
```

Comment:
this is just pulled from add traces to dataset dialog, decoupled from the button (see above) and has a  few props added

Reply:
ahh yes i w refactoring / moving in this case so didn't think of it but will do next time